src/main.rs

```diff
@@ -21,18 +23,57 @@ pub fn main() {
     }
 
     // Setup the logger
-    setup_logger().expect("setup logging");
+    {
+        #[cfg(debug_assertions)]
+        let is_debug = true;
+        #[cfg(not(debug_assertions))]
+        let is_debug = false;
+
+        let is_cli = opts.command.is_some();
+
+        // Workaround to output to console even though we compile with windows_subsystem = "windows"
+        // in release mode
+        #[cfg(target_os = "windows")]
+        {
+            if is_cli && !is_debug {
+                use winapi::um::wincon::{AttachConsole, ATTACH_PARENT_PROCESS};
+
+                unsafe {
+                    AttachConsole(ATTACH_PARENT_PROCESS);
+                }
+            }
+        }
```

Comment:
This is needed for running the release build on windows since we defined windows_subsystem = "windows" for the executable. Otherwise we don't see any output.